### ISSUE TITLE
Question: Navigating in the project using CMD/CTRL + click no longer navigate you to the class. Is it possible to fix that?

### ISSUE THREAD

**Issue by joelbrostrom** [Milad-Akarie/auto_route_library#480]
When I create a FirstPage and a SecondPage class and add them to my AppRouter and the route classes are generated.
I can now use SecondPageRoute to navigate to SecondPage, however Jumping to the declaration or implementation of SecondPageRoute does not take me to the SecondPage, but to the generated SecondPageRoute.

Similarly, if I'm working on the SecondPage class and want to see where it is navigated from I cannot click the class name and navigate to where it's referenced.
This is a mayor drawback for me since I use jump to declaration constantly.

Is there a way to set it up so that I can navigate between the actual classes instead of to ge generated ones?

**Comment by Milad-Akarie** [comment]
@joelbrostrom it is actually navigating you to the right class, Route classes are just named location that can link you to your page,  basically they're path segments on steroid. 
I find this annoying two, as I do alot of jumping.

**Comment by joelbrostrom** [comment]
Yes, it is indeed the right class technically speaking but I want to go to the "implicit" class.
I take it from your last sentence that there is no way to do this.
Have you come up with a strategy to navigate to the right class? 
Just search or can you do some clever jumping?
